v2/config/rbac/crd_manager_role.yaml

```diff
@@ -14,3 +14,15 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
```

Comment:
minor: Consider leaving a comment here that this is required for secure metrics?